include/oneapi/tbb/detail/_flow_graph_node_impl.h

```diff
@@ -63,9 +63,9 @@ class function_input_base : public receiver<Input>, no_assign {
     static_assert(!has_policy<queueing, Policy>::value || !has_policy<rejecting, Policy>::value, "");
 
     //! Constructor for function_input_base
-    function_input_base( graph &g, size_t max_concurrency, node_priority_t a_priority )
+    function_input_base( graph &g, size_t max_concurrency, node_priority_t a_priority, bool is_no_throw )
```

Comment:
What about creating an additional `IsNoThrow` template parameter for the `function_input_base` structure and therefore this field will be constexpr.

Reply:
We don't know about body at the template initialization of `function_input_base`. We know about body only in runtime.